types/accounting/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare namespace accounting {
 
     interface CurrencySettings<TFormat> {
         symbol?: string;     // default currency symbol is '$'
-        format?: TFormat;    // controls output: %s = symbol, %v = value/number
+        format?: string;    // controls output: %s = symbol, %v = value/number
```

Comment:
This change doesn't many any sense, according to the typings sometimes `format` can be a `CurrencyFormat` type.

Reply:
You are right. `format` is not always a string